javascript/ql/src/semmle/javascript/frameworks/LodashUnderscore.qll

```diff
@@ -360,6 +360,46 @@ module LodashUnderscore {
     name = "eachRight" or
     name = "first"
   }
+
+  /**
+   * A data flow step propagating an exception thrown from a callback to a Lodash/Underscore function.
+   */
+  private class ExceptionStep extends DataFlow::CallNode, DataFlow::AdditionalFlowStep {
+    ExceptionStep() {
+      exists(string name |
+        this = member(name).getACall()
+      |
+        // Collection methods
+        name = "countBy" or
+        name = "each" or
+        name = "eachRight" or
+        name = "forEach" or
+        name = "forEachRight" or
+        name = "every" or
+        name = "filter" or
+        name = "groupBy" or
+        name = "orderBy" or
+        name = "partition" or
+        name = "reduce" or
+        name = "reduceRight" or
+        name = "some" or
+        name = "sortBy" or
+
+        // Array methods
+        name = "dropRightWhile" or
+        name = "dropWhile" or
+        name = "sortedIndexBy" or
```

Comment:
There's also `sortedLastIndexBy`, apparently.

Reply:
And a bunch of others: `find`, `findIndex`, `findLast`, `findLastIndex`, `reject`, `remove`, `findKey`, `findLastKey`, `omitBy`, `pickBy`.